packages/start-frontend/src/helpers/prompt-clack.ts

```diff
@@ -0,0 +1,193 @@
+  confirm,
+  cancel,
+} from "@clack/prompts";
+import * as color from "picocolors";
+import { setTimeout as sleep } from "node:timers/promises";
+import gradient = require("gradient-string");
+
+export type UserInputTests = {
+  useVitest: boolean | undefined;
+  useStorybook: boolean | undefined;
+  useE2E: boolean | undefined;
+  useEslint: boolean | undefined;
+  usePrettier: boolean | undefined;
+};
+
+interface CLIOptions {
+  [key: string]: {
+    name: string;
+    apiSolution: { name: string; value: "graphql" | "restful" }[];
+    useModules: { name: string; value: string; label: string }[];
+  };
+}
+
+const CLIOptions: CLIOptions = {
+  react: {
+    name: "React",
+    apiSolution: [
+      { name: "RESTful", value: "restful" },
+      { name: "GraphQL", value: "graphql" },
+    ],
+    useModules: [{ name: "Add CRUD Operations", value: "crud", label: "CRUD" }],
+  },
+};
+
+// ML Colors
+const g = gradient("#53575a", "#53575a");
+const t = gradient("#53575a", "#ffff00");
+
+export async function promptClack(dir) {
+  // intro example
+  intro(
+    `${g("ꮙ START-")}${t(
+      "FRONTEND"
+    )} \n\n Welcome to the frontend setup wizard!`
+  );
+
+  // group example
+  const groupUtility = await group(
+    {
+      location: () =>
+        // text example
+        text({
+          message: color.blue("Which location you want to start project?"),
+          placeholder: "./my-app",
+          initialValue: `./${dir}` || "",
+          validate: (value) => {
+            if (!value) {
+              return "Please provide a location path";
+            }
+            // regular expression that text start from ./
+            const Reg = new RegExp("^\\./");
+            if (!Reg.test(value)) {
+              return "Please input a valid location path";
+            }
+          },
+        }),
+      jsLibrary: () => {
+        return select({
+          message: "Select a JavsScript library for UI",
+          options: Object.keys(CLIOptions).map((key) => {
+            return {
+              value: key,
+              label: CLIOptions[key].name,
+            };
+          }),
+        }) as Promise<string>;
+      },
+
+      // select example
+      apiSolution: () => {
+        return select({
+          message: "Select an API Solution (Use arrow keys)",
+          options: CLIOptions.react.apiSolution.map((solution) => {
+            return {
+              value: solution.value,
+              label: solution.name,
+            };
+          }),
+        }) as Promise<"graphql" | "restful">;
+      },
+
+      modules: (value) => {
+        return multiselect({
+          message: "Select module do you want to use (Press 'space' to select)",
+          options: CLIOptions[value.results.jsLibrary as string].useModules,
+          required: false,
+        }) as Promise<string[]>;
+      },
+
+      isUseSampleTestCode: () => {
+        return confirm({
+          message: "Add Testing codes for Catching bugs early?",
```

Comment:
There's no need to capitalize "testing" and "catching" here.

Reply:
Nvm, I'll follow what you mentioned.,